lte/gateway/c/session_manager/SessionStateEnforcer.cpp

```diff
@@ -408,19 +408,26 @@ void SessionStateEnforcer::m5g_move_to_inactive_state(
 void SessionStateEnforcer::m5g_move_to_active_state(
     std::unique_ptr<SessionState>& session, SetSmNotificationContext notif,
     SessionStateUpdateCriteria* session_uc) {
-  bool get_gnb_teid = false;
-  bool get_upf_teid = false;
-
+  bool get_gnb_teid        = false;

```

Comment:
What's the point of assigning false to `get_gnb_teid` and `get_upf_teid` here? Why not just use false directly in `update_session_rules(session, get_gnb_teid, get_upf_teid, session_uc);` ?

Reply:
Yes, directly we can pass boolean value. Fixed it.